source/develop/release-management/features/virt/vm-failover-using-vm-leases.html.md

```diff
@@ -0,0 +1,181 @@
+---
+title: VM failover using vm leases
+category: feature
+authors: nsoffer
+feature_name: VM failover using vm leases
+feature_modules: engine
+feature_status: Design
+---
+
+# VM failover using vm leases
+
+
+## Overview
+
+This feature adds the ability to start a highly available VM on another
+host when the host becomes unreachable and the host cannot be fenced.
+In the current system the VM state is unknown and the system cannot
+start the vm on one of the available hosts until the host is rebooted
+manually and the administrator mark the host as rebooted manually.
+
+This feature is tracked in bug
+https://bugzilla.redhat.com/804272
+
+## Owner
+
+- Name: [Nir Soffer](https://github.com/nirs)
+- Email: <nsoffer@redhat.com>
+
+
+## Detailed Description
+
+
+### How it works
+
+To enable this feature, the user must create a storage lease for the vm.
+Only VM holding a storage lease can use automatic failover when a host
```

Comment:
Do we want to enforce this feature? Do we want to add a lease to all VMs in e.g. 4.2 CL?